contracts/payloads/IGP105/PayloadIGP105.sol

```diff
@@ -0,0 +1,585 @@
+                baseWithdrawalLimitInUSD: 6_000_000, // $6M
+                baseBorrowLimitInUSD: 6_000_000, // $6M
+                maxBorrowLimitInUSD: 50_000_000 // $50M
+            });
+            
+            setVaultLimits(VAULT_USDe_USDC);
+
+            IFluidVaultT1(USDe_USDC_VAULT).updateLiquidationMaxLimit(LML);
+            IFluidVaultT1(USDe_USDC_VAULT).updateLiquidationThreshold(LT);
+            IFluidVaultT1(USDe_USDC_VAULT).updateCollateralFactor(CF);
+            IFluidVaultT1(USDe_USDC_VAULT).updateLiquidationPenalty(LP);
+        }
+        {
+            address USDe_USDT_VAULT = getVaultAddress(67);
+            
+            // [TYPE 1] USDe/USDT vault - Launch limits
+            VaultConfig memory VAULT_USDe_USDT = VaultConfig({
+                vault: USDe_USDT_VAULT,
+                vaultType: VAULT_TYPE.TYPE_1,
+                supplyToken: USDe_ADDRESS,
+                borrowToken: USDT_ADDRESS,
+                baseWithdrawalLimitInUSD: 6_000_000, // $6M
+                baseBorrowLimitInUSD: 6_000_000, // $6M
+                maxBorrowLimitInUSD: 50_000_000 // $50M
+            });
+            
+            setVaultLimits(VAULT_USDe_USDT);
+
+            IFluidVaultT1(USDe_USDT_VAULT).updateLiquidationMaxLimit(LML);
+            IFluidVaultT1(USDe_USDT_VAULT).updateLiquidationThreshold(LT);
+            IFluidVaultT1(USDe_USDT_VAULT).updateCollateralFactor(CF);
+            IFluidVaultT1(USDe_USDT_VAULT).updateLiquidationPenalty(LP);
+        }
+        {
+            address USDe_GHO_VAULT = getVaultAddress(68);
+            
+            // [TYPE 1] USDe/GHO vault - Launch limits
+            VaultConfig memory VAULT_USDe_GHO = VaultConfig({
+                vault: USDe_GHO_VAULT,
+                vaultType: VAULT_TYPE.TYPE_1,
+                supplyToken: USDe_ADDRESS,
+                borrowToken: GHO_ADDRESS,
+                baseWithdrawalLimitInUSD: 6_000_000, // $6M
+                baseBorrowLimitInUSD: 6_000_000, // $6M
+                maxBorrowLimitInUSD: 20_000_000 // $20M
+            });
+            
+            setVaultLimits(VAULT_USDe_GHO);
+
+            IFluidVaultT1(USDe_GHO_VAULT).updateLiquidationMaxLimit(LML);
+            IFluidVaultT1(USDe_GHO_VAULT).updateLiquidationThreshold(LT);
+            IFluidVaultT1(USDe_GHO_VAULT).updateCollateralFactor(CF);
+            IFluidVaultT1(USDe_GHO_VAULT).updateLiquidationPenalty(LP);
+        }
+    }
+
+    // @notice Action 8: Provide Credit to Team Multisig for Fluid DEX Lite
+    function action8() internal isActionSkippable(8) {
+        // Give Team Multisig 2.5M USDC credit
+        {
+            IFluidAdminDex.UserBorrowConfig[]
+                memory configs_ = new IFluidAdminDex.UserBorrowConfig[](1);
+
+            configs_[0] = IFluidAdminDex.UserBorrowConfig({
+                user: TEAM_MULTISIG,
+                token: USDC_ADDRESS,
+                mode: 1,
+                expandPercent: 50 * 1e2, // 50%
```

Comment:
should not expand, even though max debt ceiling limits let's still set expand percent as 1 and duration as max possible (see when setting paused limits)

Reply:
also, use LL interface not adminDex